packages/GenderSelection/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@mbkit/gender-selection",
+  "version": "1.0.0",
+  "source": "src/index.tsx",
+  "main": "dist/cjs/GenderSelection.js",
+  "module": "dist/esm/GenderSelection.js",
+  "types": "dist/cjs/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "../../node_modules/.bin/rollup -c ../../shared/build-tools/rollup.config.js",
+    "dev": "../../shared/build-tools/dev-package.js"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@types/classnames": "^2.2.9",
+    "@types/react": "^16.9.2",
+    "react": "^16.9.0"
+  },
+  "peerDependencies": {
+    "react": "^16.9.0"
+  },
+  "dependencies": {
+    "@mbkit/input": "^1.3.22",
+    "@mbkit/label": "^1.3.22",
+    "@mbkit/select": "^1.4.1",
+    "@mbkit/theme": "^2.2.2",
+    "classnames": "^2.2.6",
+    "sass": "^1.34.0"
```

Comment:
Let's remove `sass` as a dependency. I believe you should not need it. If you do it's only for local development, so let's move it to the `devDependencies`

Reply:
Also seeing one build error: 

> Error:   27:23  error    Don't use 'String' as a type. Use string instead  @typescript-eslint/ban-types

Reply:
https://github.com/mindbody/mbkit/pull/135/checks?check_run_id=2677745821#step:4:181

Reply:
moved scss in dev dependency and resolve build error